hazelcast/src/main/java/com/hazelcast/collection/impl/txncollection/operations/CollectionPrepareBackupOperation.java

```diff
@@ -22,31 +22,32 @@
 import com.hazelcast.nio.ObjectDataInput;
 import com.hazelcast.nio.ObjectDataOutput;
 import com.hazelcast.spi.BackupOperation;
+
 import java.io.IOException;
 
 public class CollectionPrepareBackupOperation extends CollectionOperation implements BackupOperation {
 
-    private long itemId;
-    private boolean removeOperation;
+    private long[] itemIds;
     private String transactionId;
 
     public CollectionPrepareBackupOperation() {
     }
 
-    public CollectionPrepareBackupOperation(String name, long itemId, String transactionId, boolean removeOperation) {
+    public CollectionPrepareBackupOperation(String name, long[] itemIds, String transactionId) {
         super(name);
-        this.itemId = itemId;
-        this.removeOperation = removeOperation;
+        this.itemIds = itemIds;
         this.transactionId = transactionId;
     }
 
     @Override
     public void run() throws Exception {
         CollectionContainer collectionContainer = getOrCreateContainer();
-        if (removeOperation) {
-            collectionContainer.reserveRemoveBackup(itemId, transactionId);
-        } else {
-            collectionContainer.reserveAddBackup(itemId, transactionId);
+        for (long itemId : itemIds) {
+            if (itemId > 0) {
```

Comment:
Can you hide `+/-` itemId logic into `CollectionTxnUtil`? 

like `boolean CollectionTxnUtil.isRemoved(itemId)`